content/plugin/log/index.mdx

```diff
@@ -0,0 +1,8 @@
+<!--
+This is a placeholder page.
+`docs-sidenav` expects an MDX file to exist for a /<base>
```

Comment:
Non-blocking: Probably self-evident, but as-is we'll be rendering an empty page at [`/plugin/log`](https://terraform-website-git-test-plugin-split-hashicorp.vercel.app/plugin/log).

To be totally clear: we definitely don't have to tackle in this PR (IMO), I think the `hidden` nav nodes you've got set up are great 👍

Feels like something a situation we probably want to avoid long-term for UX purposes, though definitely difficult for us to resolve on the implementation side only as I think it'd require either 1) adding content so these pages aren't empty, or 2) re-organizing & redirecting nested pages so that these empty index pages aren't necessary 🤔